### PR TITLE
Tanya class created

### PR DESCRIPTION
**Pull request by tkhoptii**

### CODE REVIEW

**Comment by Amirbek512** [comment]
great team work

**Comment by ermanyapici** [comment]
Great project :)

**Comment by tkhoptii** [comment]
thanks

File: src/tanya_project/PracticeTanya1.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package tanya_project;
+
+public class PracticeTanya1 {
```
Comment (ermanyapici) [inline src/tanya_project/PracticeTanya1.java:3]: Add main method

**Review by ermanyapici — APPROVED** [review]

**Review by Amirbek512 — APPROVED** [review]

**Review by ermanyapici — APPROVED** [review]
Approved

**Pull request merged**